sos/plugins/docker.py

```diff
@@ -29,20 +29,47 @@ class Docker(Plugin):
         ("all", "enable capture for all containers, even containers "
             "that have terminated", 'fast', False),
         ("logs", "capture logs for running containers",
-            'fast', False)
+            'fast', False),
+        ("size", "capture image sizes for ps", 'slow', False)
     ]
 
     def setup(self):
         self.add_copy_spec([
             "/var/lib/docker/repositories-*"
         ])
 
-        for subcmd in ['info', 'ps', 'ps -a', 'images', 'version']:
+        for subcmd in [
+            'info',
+            'images',
+            'network ls',
+            'ps',
+            'ps -a',
+            'stats --no-stream',
+            'version'
+        ]:
             self.add_cmd_output(
                 "{0} {1}".format(self.docker_cmd, subcmd)
             )
 
+        # separately grab ps -s as this can take a *very* long time
+        if self.get_option('size'):
+            self.add_cmd_output('{0} ps -as'.format(self.docker_cmd))
```

Comment:
What is `docker ps -as`? Is it the same as `ps -as`?

(asking, as with my `ps`, the `-s` adds signal information, not size).